index.bs

```diff
@@ -5129,6 +5130,56 @@ if the [=authenticator=] does not support [=attestation=].
 : Verification procedure
 :: Return implementation-specific values representing [=attestation type=] [=None=] and an empty [=attestation trust path=].
 
+## Apple Anonymous Attestation Statement Format ## {#sctn-apple-anonymous-attestation}
+
+This attestation statement format is exclusively used by Apple for certain types of Apple devices that support WebAuthn.
+
+: Attestation statement format identifier
+:: apple
+
+: Attestation types supported
+:: [=Anonymous CA=]
+
+: Syntax
+:: The syntax of an Apple attestation statement is defined as follows:
+
+    ```
+    $$attStmtType //= (
+                          fmt: "apple",
```

Comment:
Somewhat orthogonal (but related) to this PR and the new fmt value, the IANA registry should be updated. For references see step 19 of [section 7.1 Registering a new Credential](https://w3c.github.io/webauthn/#sctn-registering-a-new-credential), which in turn points to [IANA WebAuthn Registry](https://www.iana.org/assignments/webauthn/webauthn.xhtml). It may be best just to open a new issue for this to be done at present, then let that take place after this PR is merged.

Reply:
Created #1495 to track that

Reply:
Will resolve this review comment with the understanding that #1495 has been opened to address separately.